gh-pages/name-and-logo-usage-guidelines.md

```diff
@@ -0,0 +1,152 @@
+
+## Trademark Usage Guidelines
+
+- The Baseline Trademarks should not be used or registered in a manner that would cause confusion as to Google’s sponsorship, affiliation, or endorsement.
+- The Baseline Trademarks should not be combined with other trademarks, terms, or source identifiers.
+- Don't modify, distort or alter the Baseline Trademarks. That includes modifying the Baseline name (for example, through hyphenation, combination or abbreviation), or displaying the Baseline logos with color variations or unapproved visual elements.
+
+### Baseline name
+
+<table class="logos">
+  <tr>
+    <td>
+      <img src="assets/img/baseline-wordmark.svg" alt="Baseline wordmark"><br>
+      <a href="assets/img/baseline-wordmark.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-wordmark.svg" download>Download SVG</a>
+    </td>
+    <td>
+      <img src="assets/img/baseline-wordmark-dark.svg" class="dark-bg" alt="Baseline wordmark dark"><br>
+      <a href="assets/img/baseline-wordmark-dark.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-wordmark-dark.svg" download>Download SVG</a>
+    </td>
+  </tr>
+</table>
+
+The Baseline name should be used when talking about Baseline as a concept.
+
+### Baseline Widely Available logo
+
+<table class="logos">
+  <tr>
+    <td>
+      <img src="assets/img/baseline-widely-icon.svg" alt="Baseline widely available icon"><br>
+      <a href="assets/img/baseline-widely-icon.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-widely-icon.svg" download>Download SVG</a>
+    </td>
+    <td>
+      <img src="assets/img/baseline-widely-icon-dark.svg" class="dark-bg" alt="Baseline widely available icon dark"><br>
+      <a href="assets/img/baseline-widely-icon-dark.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-widely-icon-dark.svg" download>Download SVG</a>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="assets/img/baseline-widely-word.svg" alt="Baseline widely available wordmark"><br>
+      <a href="assets/img/baseline-widely-word.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-widely-word.svg" download>Download SVG</a>
+    </td>
+    <td>
+      <img src="assets/img/baseline-widely-word-dark.svg" class="dark-bg" alt="Baseline widely available wordmark dark"><br>
+      <a href="assets/img/baseline-widely-word-dark.png" download>Download PNG</a> |
+      <a href="assets/img/baseline-widely-word-dark.svg" download>Download SVG</a>
+    </td>
+  </tr>
+</table>
+
+The Baseline Widely Available logo should be used when talking about Baseline as a concept, or to indicate a specific web platform feature has been supported by all of the core browsers for at least 30 months, and is therefore widely interoperable.
```

Comment:
The definition we're using on the landing page is 2.5 years or 30 months, we should be consistent. We generally don't ask developers to look at the web-features package data, and I don't want to do that to them here.

Reply:
> We generally don't ask developers to look at the web-features package data

Why not? It's way easier than actually following the [formal definition](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md#interoperable-low-status) from scratch. It contains a number of catches and exceptions.

This combined with https://github.com/web-platform-dx/web-features/pull/1706/files#r1743829760 and https://github.com/web-platform-dx/web-features/pull/1706/files#r1743836903 worries me a bit. If this file isn't really under the control of web-features, then I think it might be better if one of these things happen:

- Google hosts the legalese elsewhere and we link to it. If we're not free to change it after it merges, then that would be more honest and transparent about the authorship and maintenance of the usage guidelines. We could have some sort of cheat sheet or implementers guide (that's more specifically about integrating against web-features), that links out to the legalese. 

- The text here changes to say something like "The Baseline Widely Available logo should be used when talking about Baseline as a concept, or to indicate a specific web platform feature has satisfied the status requirements described in [the Baseline status definition](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md#wider-support-high-status)."